linux/keyman-config/build.sh

```diff
@@ -72,6 +72,21 @@ build_action() {
   python3 setup.py build
 }
 
+test_action() {
+  local TEMP_DATA_DIR SCHEMA_DIR
+  TEMP_DATA_DIR=$(mktemp -d)
+  SCHEMA_DIR=$TEMP_DATA_DIR/glib-2.0/schemas
+  export XDG_DATA_DIRS=$TEMP_DATA_DIR:${XDG_DATA_DIRS-}
+  export GSETTINGS_SCHEMA_DIR="${SCHEMA_DIR}"
+  mkdir -p "$SCHEMA_DIR"
+  cp resources/com.keyman.gschema.xml "$SCHEMA_DIR"/
+  glib-compile-schemas "$SCHEMA_DIR"
+  ./run-tests.sh
+  export XDG_DATA_DIRS=${XDG_DATA_DIRS#*:}
+  unset GSETTINGS_SCHEMA_DIR
+  rm -rf "$TEMP_DATA_DIR"
```

Comment:
Nit: some vars are quoted and some aren't, also some inconsistencies in curly brace usage: `"$SCHEMA_DIR"` vs `"${SCHEMA_DIR}"`.

Reply:
done